src/Queue.js

```diff
@@ -20,33 +20,35 @@ class Queue {
     // Also sets the input process's parent queue to this queue
     // Return the newly added process
     enqueue(process) {
-        
+        this.processes.push(process);
```

Comment:
This is perfectly acceptable but another way to do it that is less lines of code is 
`process.setParentQueue(this);`
`return this.processes.push(process)`
very small thing and your way is 100% acceptable just another way to write it